Dalamud/Plugin/DalamudPluginInterface.cs

```diff
@@ -113,6 +116,16 @@ internal DalamudPluginInterface(string pluginName, PluginLoadReason reason, bool
         /// </summary>
         public DirectoryInfo DalamudAssetDirectory => Service<Dalamud>.Get().AssetDirectory;
 
+        /// <summary>
+        /// Gets the location of your plugin assembly.
+        /// </summary>
+        public FileInfo AssemblyLocation { get; }
+
+        /// <summary>
+        /// Gets the directory your plugin assembly and bundled files are stored in.
+        /// </summary>
+        public DirectoryInfo AssemblyDirectory { get; }
```

Comment:
figured i'd add it since that's the only reason someone would care about their assembly location in the first place.
`new DirectoryInfo(assemblyLocation.DirectoryName!);`

Reply:
Request complied with